app/views/pets/show.html.erb

```diff
@@ -13,9 +13,8 @@
   <%= button_to "Add to Favorites", "/favorites/#{@pet.id}", method: :patch %>
 <% end %>
 
-<%= button_to "Add to Favorites", "/favorites/#{@pet.id}", method: :patch %>
-<%= link_to "Change to #{@pet.adoption_status_text}", "/pets/#{@pet.id}/adopt", method: :patch %>
-
-<a href="/pets/<%= @pet.id %>/edit">Update Pet</a>
-<br><a href="/pets/<%= @pet.id %>" data-method="DELETE">Delete Pet</a></br></div>
+<br><%= link_to "View All Applications for #{@pet.name}", "/pets/#{@pet.id}/applications"%></br>
+<br><%= link_to "Change to #{@pet.adoption_status_text}", "/pets/#{@pet.id}/adopt", method: :patch %></br>
+<br><%= link_to "Update Pet", "/pets/#{@pet.id}/edit" %></br>
+<br><%= link_to "Delete Pet", "/pets/#{@pet.id}", method: :delete %></br></div>
```

Comment:
updated the html to use action view helper tags. Slowly getting around to changing everything over!

Reply:
Looks great! Thank you!